hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/GooglePubSubSenderProperties.java

```diff
@@ -0,0 +1,67 @@
+package pl.allegro.tech.hermes.consumers.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "google.pubsub.sender")
```

Comment:
In my opinion this section should be a child of the `consumer` section (`consumer.google.pubsub.sender`). It applies neither to `hermes-frontend` nor `hermes-management`.